.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
What does this header here do?

Reply:
It specifies the name of the template, a small description the user will see when having to pick one, and other things like default title, automatically attached labels, etc.